### DISCUSSION TITLE
Github Team Pricing

### DISCUSSION THREAD

**Discussion by gen-encrypted** [community/community#73146] [New to GitHub]
I am planning to update my account to teams tier in order to manage my private repos. I am also aiming to extend my team upon demand. How will my account will be billed? Do I have to pay 4$ monthly or my payment depends on the total invited unique users to my private repos? In addition if a user is added for one day should I have to pay a monthly fee for that user? 

Last of all, are payments done in advance or at the end of the month? Thanks for your contribution.

**Comment by YamilH** [comment]
Hey! I hope this helps you:

https://github.com/pricing
